app/main.ipynb

```diff
@@ -58,15 +82,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
-   "outputs": [],
+   "execution_count": 4,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "(21790, 10771)"
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your code for task 1 here"
+    "men_count = len(df[df[\"sex\"] == \"Male\"].index)\n",
+    "women_count = len(df[df[\"sex\"] == \"Female\"].index)\n",
+    "men_count, women_count"
```

Comment:
there is a simpler solution with [value_counts](https://pandas.pydata.org/docs/reference/api/pandas.Series.value_counts.html)